tests/Apcu/ApcuTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Tests\Apcu;
+
+/**
+ * @requires extension apc
```

Comment:
so, it looks like you don't have the apcu bc layer enabled, but that doesn't change my point: this is tested properly on travis, against apcu + apc-br-from-apcu, which means there is nothing to fix, the case is covered